packages/ui/src/components/Search.tsx

```diff
@@ -65,13 +61,15 @@ const Results = observer((props: { container: SearchContainer }) => {
 
   if (typeof result === 'string') return <ErrorMessage error={result} />;
 
-  if (result instanceof BlockInfo) {
-    const hash = encodeBase16(result.getSummary()!.getBlockHash_asU8());
-    return <Redirect to={Pages.block(hash)} />;
+  if ('block' in result) {
+    const getBlockResult = result as GetBlockResult,
+      hash = getBlockResult.block?.hash;
+    return <Redirect to={Pages.block(hash!)} />;
   }
 
-  if (result instanceof DeployInfo) {
-    const hash = encodeBase16(result.getDeploy()!.getDeployHash_asU8());
+  if ('deploy' in result) {
+    const getDeployResult = result as GetDeployResult,
+      hash = getDeployResult.deploy.hash;
```

Comment:
indent

Reply:
We are using `prettier` as formatter, and this is the default style, though very strange, intent 2 space for `,`,  but it doesn't provide an option to customize.